index.rst

```diff
@@ -143,104 +143,202 @@ Core Components
+.. imgtable::
+
+    HX711, components/sensor/hx711, hx711.jpg, Load Cell Amplifier
     Xiaomi Miscale, components/sensor/xiaomi_miscale, xiaomi_miscale.jpg
     Xiaomi Miscale2, components/sensor/xiaomi_miscale2, xiaomi_miscale2.jpg
-    ZyAura, components/sensor/zyaura, zgm053.jpg
-    Custom Sensor, components/sensor/custom, language-cpp.svg
+
+
+Magnetic
+********
+.. imgtable::
+    
+    ESP32 Hall Sensor, components/sensor/esp32_hall, magnet.svg, ESP internal
+    HMC5883L, components/sensor/hmc5883l, hmc5883l.jpg, 3-Axis Magnetometer
+    QMC5883L, components/sensor/qmc5883l, qmc5883l.jpg, 3-Axis Magnetometer
+
+
+Miscellaneous
+*************
+.. imgtable::
+
+    AS3935, components/sensor/as3935, as3935.jpg, Storm Lightning
+    Binary Sensor Map, components/sensor/binary_sensor_map, binary_sensor_map.jpg, Map Binary To Value
+    b-parasite, components/sensor/b_parasite, b_parasite.jpg, Moisture & Temp & Humid
+    EZO sensor circuits, components/sensor/ezo, ezo-ph-circuit.png, (pH)
+    Nextion, components/sensor/nextion, nextion.jpg, Sensors from display
+    Rotary Encoder, components/sensor/rotary_encoder, rotary_encoder.jpg
+    Havells Inverter, components/sensor/havells_inverter, havellsgti5000d_s.jpg, Solar Rooftop
+    Tuya Sensor, components/sensor/tuya, tuya.png
+    TX20, components/sensor/tx20, tx20.jpg, Wind Speed & Direction
+
+
+Motion
+******
+.. imgtable::
+
+    APDS9960, components/sensor/apds9960, apds9960.jpg, Colour & Gesture
+    MPU6050, components/sensor/mpu6050, mpu6050.jpg, Accelerometer & Gyroscope
+    RuuviTag, components/sensor/ruuvitag, ruuvitag.jpg, Temp & Humid & Accelerometer
+
+
+Electricity
+***********
+.. imgtable::
+
+    ADE7953, components/sensor/ade7953, ade7953.svg, Power
+    ATM90E32, components/sensor/atm90e32, atm90e32.jpg, Voltage & Current & Power
+    CSE7766, components/sensor/cse7766, cse7766.svg, Voltage & Current & Power
+    CS5460A, components/sensor/cs5460a, cs5460a.png, Voltage & Current & Power
+    CT Clamp, components/sensor/ct_clamp, ct_clamp.jpg, AC Current
+    HLW8012, components/sensor/hlw8012, hlw8012.svg, Voltage & Current & Power
+    INA219, components/sensor/ina219, ina219.jpg, DC Current
+    INA226, components/sensor/ina226, ina226.jpg, DC Current & Power
+    INA3221, components/sensor/ina3221, ina3221.jpg, 3-Ch DC Current
+    PZEM004T, components/sensor/pzem004t, pzem004t.svg, Voltage & Current & Power
+    PZEM AC, components/sensor/pzemac, pzem-ac.png, Voltage & Current & Power
+    PZEM DC, components/sensor/pzemdc, pzem-dc.png, Voltage & Current & Power
+    SDM Meter, components/sensor/sdm_meter, sdm220m.png, Modbus Energy Monitor
+    Teleinfo, components/sensor/teleinfo, teleinfo.jpg, Electrical Counter
+    Total Daily Energy, components/sensor/total_daily_energy, sigma.svg
+
+
+Environmental
+*************
+
+.. Note::
+
+    "temp" = Temperature
+
+    "humid" = Relative Humidity
+
+    "baro" = Barometric Pressure (i.e. atmospheric pressure)
```

Comment:
From what I've seen typically this will be referred to as just "pressure" in the shortened form in datasheets and the like so it seems a bit odd to choose "baro".

Reply:
changed